flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/testutils/JoinTestPrograms.java

```diff
@@ -142,7 +142,7 @@ public class JoinTestPrograms {
                         .setupTableSource(SOURCE_T1)
                         .setupTableSource(SOURCE_T2)
                         .setupTableSink(
-                                SinkTestStep.newBuilder("MySink")
+                                SinkTestStep.newBuilder("NON_WINDOW_INNER_JOIN_WITH_NULL_Sink")
```

Comment:
It is a great idea to use data warehouse like naming convention to improve the readability. It would be even better to follow the naming convention that table names commonly use lower case letters.

Reply:
makes sense, thanks
done